receivers/googlechat/googlechat.go

```diff
@@ -7,13 +7,12 @@ import (
 	"net/url"
 	"time"
 
-	"github.com/prometheus/alertmanager/template"
 	"github.com/prometheus/alertmanager/types"
```

Comment:
We're still importing the AM, right? related to [this comment](https://github.com/grafana/alerting/pull/63#discussion_r1146571465) on your other PR.